spec/factories/course_subjects.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
No needed

You would want to go to courses.rb and do

```ruby
    factory :course_with_a_subject do
      subjects { [association(:subject)] }
    end
```


https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#has_many-associations